### ISSUE TITLE
Indexer crash

### ISSUE THREAD

**Issue by gruberx** [Zondax/namadexer#56]
When indexing Namada public-testnet-14.5d79b6958580 at block 26314, the indexer crashes with the following error:

indexer log:
```
   2023-11-27T20:14:19.380581Z  INFO namadexer::database: Saving tx_update_account transaction
     at src/database.rs:635
     in namadexer::database::save_transactions
     in namadexer::database::save_block_impl with network: "public_testnet_14"
     in namadexer::database::save_block

   2023-11-27T20:14:19.381305Z ERROR namadexer::indexer: Closing block producer task due to an error saving last block: Database error: error returned from database: syntax error at or near "VALUES"
     at src/indexer/mod.rs:169

Error: DB(Database(PgDatabaseError { severity: Error, code: "42601", message: "syntax error at or near \"VALUES\"", detail: None, hint: None, position: Some(Original(183)), where: None, schema: None, table: None, column: None, data_type: None, constraint: None, file: Some("scan.l"), line: Some(1176), routine: Some("scanner_yyerror") }))
```

postgres log:
```
 2023-11-27 10:24:53.006 UTC [708] ERROR:  syntax error at end of input at character 174
 2023-11-27 10:24:53.006 UTC [708] STATEMENT:  INSERT INTO public_testnet_14.account_public_keys(
                                  update_id,
                                  public_key
                              )VALUES
```

On restart, the indexer tries to re-index the 26314 block.
The account_public_keys table was created automatically, but is empty.

Namadexer v0.1.0-rc.1
postgres (PostgreSQL) 14.10 (Debian 14.10-1.pgdg120+1)

**Comment by ainhoa-a** [comment]
This will be fixed with this PR https://github.com/Zondax/namadexer/pull/52

**Comment by neithanmo** [comment]
fixed via #52
